.github/CODEOWNERS

```diff
@@ -243,3 +243,5 @@
 /src/orbital/ @thisisdevanshu
 
 /src/fluid-relay/ @kairu-ms @necusjz @ZengTaoxu
+
+/src/azext_fleet/ @pdaru
```

Comment:
From error log,
> The following directories are missing codeowners ['/src/fleet/'].

This should be `/src/fleet/`.

Reply:
Also, in another CI failure, `Exception: No entry of fleet in service_name.json. Please add one to the file.`, please add something similar to
```
{
    "Command": "az aks",
    "AzureServiceName": "Azure Kubernetes Service (AKS)",
    "URL": "https://docs.microsoft.com/azure/aks/"
  },
```
 to `src/service_name.json`.